interface/forms/soap/templates/general_new.html

```diff
@@ -1,14 +1,44 @@
 <html>
 <head>
 <title>{xlt t='SOAP'}</title>
-{headerTemplate}
+{headerTemplate assets='jquery-ui|jquery-ui-base'}
+    {literal}
+    <script>
+        // handles the call back from the popup
```

Comment:
placemarker for ongoing code review

Reply:
"place marker for ongoing code review",

![image](https://user-images.githubusercontent.com/834958/119992918-06357400-bf99-11eb-9efb-dfe69f09c587.png)